core/block/import/importer.go

```diff
@@ -357,17 +356,16 @@ func (i *Import) getObjectID(
 	} else {
 		createdTime = time.Now()
 	}
-	if id, payload, err = i.objectIDGetter.Get(spaceID, snapshot, createdTime, updateExisting); err == nil {
-		oldIDToNew[snapshot.Id] = id
-		if snapshot.SbType == sb.SmartBlockTypeSubObject && id == "" {
-			oldIDToNew[snapshot.Id] = snapshot.Id
-		}
-		if payload.RootRawChange != nil {
-			createPayloads[id] = payload
-		}
-		return nil
+	if idGetter, err := i.idGetterProvider.ProvideIdGetter(snapshot.SbType); err != nil {
```

Comment:
Maybe format code like this:
```
	idGetter, err := i.idGetterProvider.ProvideIdGetter(snapshot.SbType)
	if err != nil {
		return err
	}
	id, payload, err = idGetter.GetID(spaceID, snapshot, createdTime, updateExisting)
	if err != nil {
		return err
	}
```


if-else here looks like alternative action, but it's a chain of actions